Cargo.toml

```diff
@@ -6,4 +6,4 @@ authors = ["dt.ro <dt.ro@canaltp.fr>"]
 [dependencies]
 log = "0.4"
 osmpbfreader = "0.11"
-geo = "0.7.4"
```

Comment:
isn't it better to put `>0.6` ?

Reply:
yes, right

Reply:
hum, maybe not ... seems ok here, but Cosmogony use 0.7.4 version with that syntax.